traefik.d/local.yml

```diff
@@ -0,0 +1,50 @@
+# Proxy the back-end and front-end running on the local host
```

Comment:
Fair enough, but yet another file/folder in the root...

Reply:
Are you suggesting a better location?